hyperspy/_signals/lazy.py

```diff
@@ -168,6 +170,83 @@ def rechunk(self,
                                                                   **kwargs)
                                                 )
 
+    def _repr_html_(self):
+        table = self._repr_html_table()
```

Comment:
This is nice, but I think that it should be done in a separate PR.
I suspect that it could be simplified using https://github.com/dask/dask/pull/8019.

Reply:
Thanks for pointing this out!  I'll add this in as a different PR and maybe see if there are other places where better html representation can be added into hyperspy.